using-the-grid/local-grid-proxy.md

```diff
@@ -0,0 +1,82 @@
+# Creating a grid proxy without LbScripts
+
+Sometimes it is useful to create a grid proxy when LbScripts (and therefore `lhcb-proxy-init`) is unavailable.
+The most commonly use for this is to be able to access data at grid sites using XRootD.
+
+Fortunately a grid proxy can be easily created using [voms](https://italiangrid.github.io/voms/index.html) as follows.
+
+## Installation
+
+The voms client can be installed in the following ways, if your operating system isn't listed see [here](https://github.com/italiangrid/voms-clients) (and consider [making a pull request](../CONTRIBUTING.html) to update this guide).
+
+#### macOS with [homebrew](https://brew.sh)
+
+```bash
+brew cask install java
+brew cask install jce-unlimited-strength-policy
+brew install voms
+```
+
+#### Ubuntu
+
+```bash
+sudo apt-get install voms-clients
+```
+
+#### CentOS
+
+```bash
+sudo yum install epel-release
+sudo yum install voms-clients
+```
+
+#### Arch Linux
+
+Install [voms from the AUR](https://aur.archlinux.org/packages/voms/).
```

Comment:
lol (no commands for this one? 😉)

Reply:
Using makepkg or a wrapper? If the latter, [which one...](https://wiki.archlinux.org/index.php/AUR_helpers) 😆